sound/soc/sof/intel/hda.h

```diff
@@ -392,6 +392,9 @@ struct sof_intel_dsp_bdl {
 #define SOF_HDA_PLAYBACK		0
 #define SOF_HDA_CAPTURE			1
 
+/* time in ms for opportunistic D0I3 entry delay */
+#define SOF_HDA_D0I3_WORK_DELAY_MS	5000
```

Comment:
Isn't entering D0i3 rather light-weight? If so, we could do that already after a second or 2 of idle time?

Reply:
The idea is that the delay to attempt entering D0i3 in S0 should be longer than it takes for the runtime suspend to kick in. runtime suspend delay is 3s. So if the DSP is not in runtime suspend, a few seconds later, you'd attempt to put it in D0i3 if possible